src/app/core/activity/ccd-activity/ccd-activity-icon/ccd-activity-icon.component.css

```diff
@@ -14,27 +14,15 @@
     padding: 5px 0px;
     position: absolute;
     z-index: 1;
-    top: -65%;
-    right: 100%;
+    /* top: -65%; */
+    /* right: 100%; */
     /*bottom: 125%;*/
```

Comment:
I think the consensus is that if you do not need the code, simply remove it.  It is in git history should you need to retrieve it.   The code is easier to read with less noises